idea/testData/checker/LocalObjects.kt

```diff
@@ -1,4 +1,3 @@
-// FIR_COMPARISON
   object A {
```

Comment:
The regression is due to the `DEBUG` error at line 21. I am not sure how we should fix this. Maybe we skip reporting of such "debug" issues that are never highlighted in FE1.0?

Reply:
This is now handled by the first commit in this PR, where we maintain a separate set of test files for FIR test cases.